openapi/api.yml

```diff
@@ -678,3 +678,98 @@ paths:
             items:
               type: string
         required: true
+  '/flags_v2/{application_id}':
```

Comment:
ok we have `flag_v2` endpoint now. Should we modify assessment flag endpoints to these?

Reply:
Yes to use the new flag stuff you'll need to use flags_v2. Have left the old stuff there for now so stuff doesn't fallover too much during testing and so we can decide on a migration.,